plugins/kotlin/idea/tests/testData/idea/actions/kotlinAddImportAction/MultipleMethodsAvailable.kt

```diff
@@ -0,0 +1,10 @@
+// EXPECT_VARIANT_IN_ORDER public fun bar(): kotlin.Unit defined in root.q
```

Comment:
It would like to suggest to wrap a string with quotes, so smth like
```
EXPECT_VARIANT_IN_ORDER "public fun bar(): kotlin.Unit defined in root.q"
```

Reply:
Could you please add another instruction to check that deprecated method is not present here ?

Reply:
That expectation doesn't make sense in this file, since the deprecated method is displayed (just displayed last). But I have indeed added such an instruction to OverloadsPresent.kt, which represents that scenario.